modules/editor/editor.js

```diff
@@ -1,5 +1,6 @@
 export default class Editor {
-	constructor( id, settings ) {
-		document.getElementById( id ).innerHTML = settings.content;
+	constructor( id ) {
```

Comment:
This assumes we already mounted the post content in the DOM. I think it's better to get a string as input.